src/main/kotlin/us/timinc/mc/cobblemon/spawnnotification/util/PlayerUtil.kt

```diff
@@ -0,0 +1,37 @@
+package us.timinc.mc.cobblemon.spawnnotification.util
+
+import com.cobblemon.mod.common.util.server
+import net.minecraft.registry.RegistryKey
+import net.minecraft.server.network.ServerPlayerEntity
+import net.minecraft.util.math.BlockPos
+import net.minecraft.world.dimension.DimensionType
+import kotlin.math.sqrt
+
+object PlayerUtil {
+    fun getValidPlayers(
+        pos: BlockPos,
+        range: Int,
+        dimensionKey: RegistryKey<DimensionType>,
+        playerLimit: Int
```

Comment:
Four arguments is something I'm definitely not happy with as a long-term solution... I guess I could pass in a `World` instead of `pos` and `dimensionKey` separately, but I also don't really like that either. Any particular thoughts, or just leave it for now?